integ-test/src/test/resources/indexDefinitions/nested_simple_index_mapping.json

```diff
@@ -21,6 +21,14 @@
                 "ignore_above": 256
               }
             }
+          },
+          "moveInDate" : {
```

Comment:
did we want to test this with a `nested` type too?

Reply:
This is a inner object to the nested object `address`. We have IT tests to test with the `nested` function in  [NestedIT.java#L373-L400](https://github.com/opensearch-project/sql/blob/71147ffc14ff7460f420d083a92b707481570e10/integ-test/src/test/java/org/opensearch/sql/sql/NestedIT.java#L373-L400). Is this what you were thinking of testing?

Reply:
Yes.  I think I was wondering if we had nested within nested.

Reply:
Yes we already have tests for nested in nested too.
[NestedIT.java#L106-L118](https://github.com/opensearch-project/sql/blob/main/integ-test/src/test/java/org/opensearch/sql/sql/NestedIT.java#L106-L118)